backend/src/services/write/locality.ts

```diff
@@ -37,6 +37,10 @@ export const writeLocality = async (
 
   locality.hominin_skeletal_remains = getHomininSkeletalRemains(locality)
 
+  locality.bipedal_footprints = !!locality.bipedal_footprints
+  locality.stone_tool_cut_marks_on_bones = !!locality.stone_tool_cut_marks_on_bones
+  locality.stone_tool_technology = !!locality.stone_tool_technology
```

Comment:
The previous comment meaning these variables.

Reply:
is false not converted to false? is it not negation of negation? There indeed was an error that occurred with null. This would be a lighter fix compared to defining default values to front although I agree that that would be more correct approach.

Reply:
Checked as well, false turns true. Needs more work

Reply:
`!!false` is `false` but `!!"false"` is `true`

Reply:
now it correctly handles what our front puts out and turns rest to false